src/compiler/crystal/semantic/flags.cr

```diff
@@ -39,6 +39,7 @@ class Crystal::Program
       flags.add "freebsd"
       flags.add "freebsd#{codegen_target.freebsd_version}"
     end
+    flags.add "dragonfly" if codegen_target.dragonfly?
     flags.add "openbsd" if codegen_target.openbsd?
```

Comment:
```suggestion
    flags.add "openbsd" if codegen_target.openbsd?
    flags.add "dragonfly" if codegen_target.dragonfly?

    flags.add "bsd" if codegen_target.bsd?
```

Reply:
The actual code to add the `bsd` flag here seems to have been lost, but there's no need at all to suggest reordering terms.

Reply:
We can't make the changes to use ```flag?(:bsd)``` yet. That does not work, it breaks at least the FreeBSD build! We first have to add the ```flags.add "bsd" ...``` code, then bootstrap the compiler with it, and only the bootstrapped compiler is able to build code that contains ```flag?(:bsd)``` code. If we add it now, it will break all BSD platforms :(. I suggest adding the bsd flag in semantic/flags.cr, but revert the parts of commit 1181ab4 that use flag bsd.

Reply:
This wasn't caught by CI because there is no FreeBSD or OpenBSD CI.